code/lib/api/src/shortcut.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/shortcut';
```

Comment:
Should this be `shortcut.d.ts`?

Reply:
nope. that file does not exist... this points to the implementation.

I just didn't want to move the file and deal with that.